kalite/distributed/management/commands/screenshots.py

```diff
@@ -226,10 +227,19 @@ def make_filename(self, slug):
         filename = "%s/%s%s" % (self.output_path, slug, settings.SCREENSHOTS_EXTENSION)
         return filename
 
-    def snap(self, slug):
+    def snap(self, slug, focus):
         filename = self.make_filename(slug=slug)
         self.loginfo('====> Snapping %s --titled-- "%s" --> %s%s ...' %
                  (self.browser.current_url, self.browser.title, slug, settings.SCREENSHOTS_EXTENSION))
+
+        if focus:
+            # Apply the specified styles to element. Currently only selection by
+            # id is supported. TODO: Extend it a more generic CSS selector.
+            element_id = focus['element_id']
+            styles = focus['styles']
+            for key, value in styles.iteritems():
+                self.browser.execute_script('document.getElementById("%s").style.%s = "%s"' % (element_id, key, value))
```

Comment:
Be careful with this -- it could be sensitive to the order in which iteritems returns elements! On my test it doesn't properly change the `borderColor`.